lib/cc/services/asana.rb

```diff
@@ -54,7 +54,7 @@ def receive_vulnerability
 
 private
 
-  def create_task(name, notes = nil)
+  def create_task(name, notes = "(No details available)")
```

Comment:
Have you tried a blank string here instead? Does that also fail? Remembering the error, I thought it was more about the type of the attribute rather than content.